js/msp/MSPHelper.js

```diff
@@ -957,7 +958,7 @@ MspHelper.prototype.process_data = function(dataHandler) {
             dataHandler.callbacks.splice(i, 1);
 
             // fire callback
-            if (callback) callback({'command': code, 'data': data, 'length': data.byteLength});
+            if (callback) callback({'command': code, 'data': data, 'length': data.byteLength, 'crcError': crcError});
```

Comment:
But frankly all of the callbacks and handlers for the msp messages are simply expecting the data object to be valid.  So any crc error regardless of the message type would case a runtime error.  It's just that crc errors are apparently rare with the exception of the new jumbo-packets.  This probably explains the very occasional Configurator hangs I've seen in the past.  Probably a crc error not being handled for some message resulting in a javascript runtime error.  Ideally every message handler *should* check for the new crcError flag and act appropriately but not sure if it's worth the effort.